### PR TITLE
Preston Patel Arrested

### PR DESCRIPTION
**Pull request by rhicora**
- Multiplies sensation value for famous people who get arrested

### CODE REVIEW

**Comment by GeneriedJenelle** [comment]
image

File: code/modules/paperwork/photography_journalism.dm [hunk 1]
```diff
@@ -29,7 +29,17 @@
 			if(A.mind.prefs)
 				if(A.mind.prefs.economic_status == CLASS_UPPER)
 					sensational += 5
-
+					
+			// If this person is unimportant, we don't care if they were arrested
+			// If they're in an important role or upper class, check to see if they look arrested.
+			
+			if(sensational > 0) 
+				var/list/clothing_list = A.worn_clothing
```
Comment (GeneriedJenelle) [inline code/modules/paperwork/photography_journalism.dm]: Here's a cleaner/less CPU intensive way of doing this. Each human (check under VV) has a variable called "w_uniform" which is always updated to their current worn uniform.  Also take note this code will not work if someone (somehow) changes the name of the orange jumpsuit, say, uses a hand labeller or something. This is where the istype proc would be useful.

**Code can look a bit like this:**

`if(A.w_uniform)
	var/obj/item/clothing/C = A.w_uniform
	if (istype(C, /obj/item/clothing/under/color/orange))
		sensational = sensational * 2
`

**Comment by Myoblongvagooglia** [comment]
Copyright infringement!!!!!

File: code/modules/paperwork/photography_journalism.dm (hunk 1 above)
Comment (rhicora) [reply code/modules/paperwork/photography_journalism.dm]: Updated with the new code!

**Pull request merged**